dynect/src/main/java/denominator/dynect/DynECTErrorDecoder.java

```diff
@@ -76,6 +76,9 @@ public Exception decode(String methodKey, Response response) {
         for (Message message : messages) {
           if ("token: This session already has a job running".equals(message.info())) {
             return new RetryableException(messages.toString(), cause, null);
+          } else if ("login: Bad or expired credentials".equals(message.info)) {
```

Comment:
@adriancole sure -- just pushed 3a6ccd60a15cc0ac1bca09903d8abfc91cf07389 to address that